6.x/crud-buttons.md

```diff
@@ -150,31 +151,35 @@ CRUD::button('email')->stack('line')->view('crud::buttons.quick')->meta([
     'ajax' => true, // <- just add `ajax` and it's ready to make ajax request
 ]);
 
-// customize Ajax URL, method, success/error messages with optional attributes
-CRUD::button('email')->stack('line')->view('crud::buttons.quick')->meta([
-    'access' => 'email', // operation name or boolean
-    'label' => 'Email',
-    'icon' => 'la la-envelope',
-    'wrapper' => [
-        'href' => function ($entry, $crud) {
-            return backpack_url("pet-shop/invoice/$entry->id/email");
-        },
-    ],
-    'ajax' => [
-        'method' => 'POST',
-        //'refreshCrudTable' => true, // to refresh table on success
-        'success_title' => "Payment Reminder Sent", // configure messages here if not receiving in server response
-        'success_message' => 'The payment reminder has been sent successfully.',
-        'error_title' => 'Error',
-        'error_message' => 'There was an error sending the payment reminder. Please try again.',
-    ],
-    /*
-     * Server Response examples:
-     * Success response; ex: return ["message" => "Email sent successfully"];
-     * OR
-     * Laravel Error response; ex: abort(500); abort(403, 'Unauthorized Access!');
-     */
-]);
+// optinal ajax configuration
+'ajax' => [
+    'method' => 'POST',
+    'refreshCrudTable' => false, // should the crud table be refreshed after a successful request ?
```

Comment:
We should comment `refreshCrudTable`. In between copy-paste, people may miss disabling this.